frontend/src/store/slice/Experiments/ExperimentsSelectors.ts

```diff
@@ -90,3 +90,11 @@ export const selectExperimentFunctionStatus =
 export const selectExperimentFunctionHasNWB =
   (uid: string, nodeId: string) => (state: RootState) =>
     selectExperimentFunction(uid, nodeId)(state).hasNWB
+
+export const selectFrameRate = (nodeId: string | null) => (state: RootState) => {
+  if(!nodeId) return 50
+  const list = selectExperimentList(state)
```

Comment:
specify uid required.
nodeId can be same value among experiments.

e.g.)
create pipeline -> RUN_ALL
then just change frame rate and RUN_ALL
two workflows with different uid, but nodeID is not changed.

Reply:
@ReiHashimoto I have fixed it, please check again

Reply:
@sanglevinh 
Looks fine, thank you.